graphbolt/include/graphbolt/random.h

```diff
@@ -0,0 +1,67 @@
+// Add for RandomEngine.
+#include <dmlc/thread_local.h>
```

Comment:
We should protect this header by #ifndef ....
e.g.:
https://github.com/dmlc/dgl/blob/69d9b726549a22566a6f78ebd14d1b5b8810adb4/graphbolt/include/graphbolt/csc_sampling_graph.h#L6

Reply:
Get it